swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -614,10 +614,10 @@ function getEmailContent(int appointmentNumber, Appointment appointment, Payment
 
 You can run this service by navigating to the project root and using the `bal run` command.
 
-```
-send-emails-from-a-service$ bal run
+```bash
```

Comment:
@MaryamZi Should this be indented too?

Reply:
Yeah, Anupama's suggestion was indent code blocks that were under bullet/numbered points.

Reply:
My bad, this is a heading, right? Sorry, we don't need indentation here.